Agreements/Internal/TCNW_C.txt

```diff
@@ -1,22 +1,20 @@
-COPX:EGHI:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:WCO
-COPX:EGHH:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:WCO
-COPX:EGLF:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:WCO
-COPX:EGLK:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:WCO
-COPX:EGTF:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:WCO
-COPX:EGHL:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:WCO
+COPX:EGHI:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
+COPX:EGHH:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
+COPX:EGLF:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
+COPX:EGLK:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
+COPX:EGTF:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
+COPX:EGHL:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
 
-COPX:EGLL:*:WOBUN:*:*:London TC NW:London AC Daventry:14000:*:WOBUN
-COPX:EGLL:*:BUZAD:*:*:London TC NW:London AC Daventry:14000:*:BUZAD
-COPX:EGLL:*:UMLAT:*:*:London TC NW:London AC Daventry:14000:*:UMLAT
```

Comment:
I suggest retaining UMLAT and ULTIB - in case someone files UMLAT DCT TNT ... and then we have no agreements for them

Reply:
Sure thing!

Reply:
Thinking about it, by the time the aircraft passes UMLAT/ULTIB, the agreement will disappear. So it might be more useful just not to define the point and simply specify the level.